jumanji/training/evaluator.py

```diff
@@ -87,6 +87,9 @@ def body_fun(
                 lambda x: x[None], acting_state.timestep.observation
             )
             action = acting_policy(observation, action_key)
```

Comment:
Remove comments?

Reply:
Forgot to remove those after debugging. Thanks @DriesSmit they've been removed.